projects/storefrontlib/src/shared/components/item-counter/item-counter.component.ts

```diff
@@ -211,6 +218,13 @@ export class ItemCounterComponent
     return this.value >= this.max || this.value <= this.min;
   }
 
+  onInputBlur(): void {
```

Comment:
Perhaps we can call this function "validateValueOrDefault" or something along those lines?